src/pages/markets/Markets.tsx

```diff
@@ -91,14 +114,62 @@ const $HeaderSection = styled.section`
     margin-bottom: 1rem;
   }
 `;
+
+const $MarketsPageBanner = styled.div`
+  ${layoutMixins.row}
+  height: 5rem;
+  border-radius: 10px;
+  background-color: var(--color-layer-4);
+  margin-bottom: 1rem;
+  padding: 0 1.5rem;
+  justify-content: space-between;
+  gap: 0.5rem;
+  position: relative;
+  overflow: hidden;
+
+  span {
+    font: var(--font-medium-medium);
+  }
+
+  @media ${breakpoints.desktopSmall} {
+    margin-left: 1rem;
+    margin-right: 1rem;
+  }
+
+  @media ${breakpoints.tablet} {
+    span,
+    button {
+      z-index: 1;
+    }
+  }
+`;
+
+const $FlagOverlay = styled.div`
+  width: 573px; // width of flag image
```

Comment:
can we do comments in template strings? I thought this broke stuff

Reply:
It doesn't seem to be messing with the other css props

Reply:
Moved the comment to above $FlagOverlay